Ex08.R

```diff
@@ -0,0 +1,41 @@
+# Exercise 08
+setwd("/Users/bethoceguera/Documents/PhD Year 1/First Semester/Introduction to Biocomputing/Exercise08")
+
+# 1. Write R code that replicates the functionality of the head function we used in bash
+data <- read.csv("iris.csv")
+data[1:6,]
```

Comment:
The assignment asked you to set a variable to the number of lines to be extracted. This is missing here

Reply:
-0.25